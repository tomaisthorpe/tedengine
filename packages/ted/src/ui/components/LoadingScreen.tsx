import styled from 'styled-components';

const Container = styled.div`
  position: absolute;
  top: 0;
  left: 0;
  background: rgba(51, 51, 102, 1);
  width: 1024px;
  height: 768px;
  font-family: sans-serif;
  color: white;
  display: flex;
  justify-content: center;
  align-items: center;
  text-transform: uppercase;
  font-size: 12px;
  font-weight: bold;
`;

export default function LoadingScreen() {
  return <Container>Loading...</Container>;
}
