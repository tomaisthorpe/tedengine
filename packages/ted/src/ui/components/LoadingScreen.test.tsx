import { render } from '@testing-library/react';
import { LoadingScreen } from './LoadingScreen';

test('renders LoadingScreen component', () => {
  const { container } = render(
    // eslint-disable-next-line @typescript-eslint/no-empty-function
    <LoadingScreen />,
  );
  expect(container).toMatchSnapshot();
});
