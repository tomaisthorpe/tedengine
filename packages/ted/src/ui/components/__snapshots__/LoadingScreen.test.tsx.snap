// Jest Snapshot v1, https://jestjs.io/docs/snapshot-testing

exports[`renders LoadingScreen component 1`] = `
<div>
  <div
    class="sc-bRKDuR eimafO"
  >
    Loading...
  </div>
</div>
`;
